deepchem/feat/tests/test_smiles_tokenizer.py

```diff
@@ -0,0 +1,21 @@
+# Requirements - transformers, tokenizers
+import os
+from unittest import TestCase
+from deepchem.feat.smiles_tokenizer import SmilesTokenizer
+from transformers import RobertaForMaskedLM
+
+
+class TestSmilesTokenizer(TestCase):
+  """Tests the SmilesTokenizer to load the USPTO vocab file and a ChemBERTa Masked LM model with pre-trained weights.."""
+
+
+  def test_tokenize(self):
+      current_dir = os.path.dirname(os.path.realpath(__file__))
+      vocab_path = os.path.join(current_dir, 'data', 'vocab.txt')
+
+      model = RobertaForMaskedLM.from_pretrained('seyonec/SMILES_tokenized_PubChem_shard00_50k')
+      model.num_parameters()
+
+      tokenizer = SmilesTokenizer(vocab_path, max_len=model.config.max_position_embeddings)
+      print(tokenizer.encode("CCC(CC)COC(=O)[C@H](C)N[P@](=O)(OC[C@H]1O[C@](C#N)([C@H](O)[C@@H]1O)C1=CC=C2N1N=CN=C2N)OC1=CC=CC=C1"))
```

Comment:
Could you add some assertions into this test? It would be good to add checks that the tokenizer has ocrrect behavior and have assertions to guard it

Reply:
New assert looks good! Could you remove the print statement now that we have the assert in?